blogposts/2021/api-documentation-generated-for-all-your-code.md

```diff
@@ -0,0 +1,108 @@
+
+## On by default, for everyone
+
+The most important announcement here is that API docs is now an on-by-default experiment for everyone! You can begin trying it out on Sourcegraph.com as well as your own Sourcegraph instance.
+
+<figure>
+  <img height="200px" src="https://sourcegraphstatic.com/blog/api-docs/api-docs-button.png" alt="The new API Docs button on repository pages">
+  <figcaption>Use the API docs button on any repository page to access API docs</figcaption>
+</figure>
+
+<figure>
+  <img height="80px" src="https://sourcegraphstatic.com/blog/api-docs/welcome-prompt.png" alt="The new welcome prompt describing how to use API docs">
+  <figcaption>Once you click it, you'll be greeted with information about how to get started</figcaption>
+</figure>
+
+### Understands the types of symbols, and uses iconography to communicate
+
+API docs is built on code intelligence data, so it has a deep semantic understanding of symbols, their type information, whether they're deprecated, a function, variable, interface, etc., and the UI now communicates this information using icons:
+
+<figure>
+  <img src="https://sourcegraphstatic.com/blog/api-docs/symbols.gif" alt="A structured tree of symbols with type information to navigate a Go package">
+  <figcaption>A structured tree of symbols with type information to navigate a Go package's documentation</figcaption>
+</figure>
+
+### A deep understanding of code
+
+We're paying attention to the small details, and really tailoring API docs to each language.
+
+At Sourcegraph, we use primarily Go and TypeScript—so we're starting by building an experience that Go developers will love, then we'll expand to more languages and build an experience that is really tailored to, and makes sense in the context of, the individual language.
+
+One way we've enabled this is by allowing LSIF language indexers to emit data in the way they see fit. Sourcegraph merely renders it, indexes it, and adds a nice navigation experience.
+
+<figure>
+  <img src="https://sourcegraphstatic.com/blog/api-docs/url-hash-displays.gif" alt="Browser URL bar showing a URL and hash designed specifically for the Go language">
+  <figcaption>The Go LSIF indexer chooses to display URL hashes in a way that makes sense specifically for Go, with a `Receiver.Method` format</figcaption>
+</figure>
+
+<figure>
+  <img height="400px" src="https://sourcegraphstatic.com/blog/api-docs/symbol-types.gif" alt="Hovering over symbols and showing Sourcegraph's deep semantic understanding of type information">
+  <figcaption>Hovering over symbols shows Sourcegraph's deep semantic understanding of type information, which you'll soon be able to search and filter based on.</figcaption>
```

Comment:
@slimsag The caption ends quite abruptly, is there a way you can reword it? It might just be the ending with "on" which always sounds a bit awkward 😅 

I've also edited the other captions so they're consistent with no periods at the end.

Reply:
```suggestion
  <figcaption>Soon it will be possible to search and filter using Sourcegraph's deep semantic understanding of type information</figcaption>
```